projects/storefrontlib/src/cms-components/product/product-list/container/product-list-component.service.ts

```diff
@@ -37,8 +37,8 @@ export class ProductListComponentService {
 
   protected sub: Subscription;
 
-  protected readonly RELEVANCE_CATEGORY = ':relevance:category:';
-  protected readonly RELEVANCE_BRAND = ':relevance:brand:';
+  protected readonly RELEVANCE_CATEGORY = ':relevance:allCategories:';
+  protected readonly RELEVANCE_BRAND = ':relevance:allCategories:';
```

Comment:
we should get ride of this specific constant, as we rely now on a single SOLR index property.

Reply:
Change committed